README.md

```diff
@@ -22,7 +22,7 @@ git clone https://github.com/Chemaclass/chemaclass.com.git
 
 2. Start the project
 ```
-zola serve
+npm start
```

Comment:
How so? 🤔

Reply:
It's all about convenience. You can still execute `zola serve` as always, and it will work perfectly fine. However, in JavaScript projects, using `npm` scripts has become the standard.

The main benefit is reducing friction for developers, especially when switching between projects. In JS/TS projects, it's common to define scripts like `start` to ensure a consistent entry point. Without this, developers often have to search for the exact command each project requires. Here are some examples of common commands:

**Angular:**  `ng serve`
**NextJS:** `next dev`
**Electron:** `npx electron .`
**Vite:** `vite` 

By wrapping commands in a `start` script, you simplify onboarding for yourself or others. It’s not mandatory, but it reduces cognitive load and makes it easier to jump between projects with different frameworks or tools. 😊

Reply:
Ok for me, thanks for clarifying 👍🏼